tests/test_settings.py

```diff
@@ -1,6 +1,7 @@
 """
 Validate that our settings functions work, and we can create yaml files
 """
+
```

Comment:
Weird, modernize doesn't suggest that it is needed.  I can look into fixing that in another PR since I wrote the tests.  It looks like we are missing a bunch of absolute imports anyway.